.github/workflows/release.yml

```diff
@@ -13,23 +13,57 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
-          ref: main
+          ref: elkezza-sdk-versioning  # Change 'main' before the merge
           fetch-depth: 0
 
+
+      - name: Install xmlstarlet
+        run: sudo apt-get update && sudo apt-get install -y xmlstarlet
+
+      - name: Get tag name
+        run: |
+          TAG_NAME=${GITHUB_REF#refs/tags/}
+          echo "TAG_NAME=$TAG_NAME"
+          echo "TAG_NAME=$TAG_NAME" >> $GITHUB_ENV
+          
+      - name: Update version numbers in files
+        run: |
+          BASE_VERSION=${TAG_NAME#v}
+          echo "Base version: $BASE_VERSION"
+          xmlstarlet ed --inplace --update "//*[local-name()='artifactVersion']" --value "$BASE_VERSION" pom.xml
+
+          sed -i "s/<version>.*<\/version>/<version>$BASE_VERSION<\/version>/g" README.md
+          sed -i "s/'com.exoscale.sdk:sdk:.*'/'com.exoscale.sdk:sdk:$BASE_VERSION'/g" README.md
+
+          
+          sed -i "s/<version>.*<\/version>/<version>$BASE_VERSION<\/version>/g" sdk/README.md
+          sed -i "s/'com.exoscale.sdk:sdk:.*'/'com.exoscale.sdk:sdk:$BASE_VERSION'/g" sdk/README.md
+
+
+          CONFIG_FILE="sdk/src/main/java/com/exoscale/sdk/client/Configuration.java"
+          sed -i "s/public static final String VERSION = \".*\";/public static final String VERSION = \"$BASE_VERSION\";/g" "$CONFIG_FILE"
+
+          git config user.name "Automated"
+          git config user.email "actions@users.noreply.github.com"
```

Comment:
```suggestion
          git config user.name "Exoscale"
          git config user.email "operation+build@exoscale.net"
```
@pierre-emmanuelJ do you think this works?

Reply:
I tested it and it works : ) 
![image](https://github.com/user-attachments/assets/724721ce-2368-41cf-8262-e2dbbb5f258e)
big thanks for this suggestion!